administrator/language/de-DE/com_fields.ini

```diff
@@ -22,7 +22,6 @@ COM_FIELDS_FIELD_EDITABLE_IN_BOTH="Beide"
 COM_FIELDS_FIELD_EDITABLE_IN_LABEL="Einsatzbereich"
 COM_FIELDS_FIELD_EDITABLE_IN_SITE="Website"
 COM_FIELDS_FIELD_FORM_EDIT="Feld bearbeiten"
-COM_FIELDS_FIELD_FORM_LAYOUT_LABEL="Layout"
```

Comment:
Still used in core (will later create a PR):
https://github.com/joomla/joomla-cms/blob/049579c880fcb7a669404b83a35f268c84188239/plugins/fields/subform/params/subform.xml#L70

Reply:
Done: https://github.com/joomla/joomla-cms/pull/42584